doc/develop/sca/codechecker.rst

```diff
@@ -42,6 +42,23 @@ To configure CodeChecker or analyzers used, arguments can be passed using the
     -DCODECHECKER_ANALYZE_OPTS="--config;$CODECHECKER_CONFIG_FILE;--timeout;60"
 
 
+Storing CodeChecker results
+***************************
+
+If a CodeChecker server is active, the results can be uploaded using the optional
+``CODECHECKER_STORE`` or ``CODECHECKER_STORE_OPTS`` parameters, e.g.
```

Comment:
I don't see any description of `CODECHECKER_STORE` itself.

Users will have to examine CMake code to realize that this var must be a true value.
Also it seems that if only using `CODECHECKER_STORE=y`, then additional arguments must be specified by using `CODECHECKER_ANALYZE_OPTS`, but from this conversation https://github.com/zephyrproject-rtos/zephyr/pull/61693/commits/88e9a5ebb74c8a520635dbc8ec51c014812c12d6#r1300004774 i'm not fully sure.

Can you extend the description so that it's clearer what is required and how the behavior will be if only using: `CODECHECKER_STORE=y`.